### ISSUE TITLE
Offspring do not recuit parents

### ISSUE THREAD

**Issue by mariopaolucci** [LABSS/PROTON-OC#255]
Due to

https://github.com/LABSS/PROTON-OC/blob/09f74038adadee13dfb61d3264fe0171a2db6612/PROTON-OC.nlogo#L1290-L1293

having a distance check, the reverse-link inclusion in the turtle set is vane because offspring have infinite distance to their parents (the distance calculation does not allow going counter-flow).

This implies that potential recruitment of parents from offspring cannot happen.

**Comment by mariopaolucci** [comment]
Context information coming from testing in the 

```
observer: "from"
observer> test2
observer>  ask turtle 9 [ nw:with-context turtles person-links [ show sort [who] of (turtle-set nw:turtles-in-radius 1 nw:turtles-in-reverse-radius 1) with [ nw:distance-to myself = 1 ] ] ]
(person 9): [7 8 10]
observer>  ask turtle 7 [ nw:with-context turtles person-links [ show sort [who] of (turtle-set nw:turtles-in-radius 1 nw:turtles-in-reverse-radius 1) with [ nw:distance-to myself = 1 ] ] ]
(person 7): [8]
observer> ask turtle 9 [ nw:with-context turtles person-links [ show nw:distance-to turtle 7 ] ]
(person 9): 2
observer> ask friendship-link 8 9 [ die ]
observer> ask turtle 9 [ nw:with-context turtles person-links [ show nw:distance-to turtle 7 ] ]
(person 9): false
```

This happens using test2 from here:

https://github.com/LABSS/PROTON-OC/blob/63ff9e648661540db982b5b29e064a2caf4f8ca4/nls_files/make-network.nls#L1-L54

**Comment by mariopaolucci** [comment]
closed by 5c04a16a7e267962ea4fd1f96dd10195a5e66860
